src/validator.ts

```diff
@@ -42,7 +42,7 @@ type HaveArgs = Method<Linked> | Constructor<Linked>
 type notEmpty = Program<Linked> | Test<Linked> | Method<Linked>
 
 const canBeCalledWithArgs = (member1: HaveArgs, member2: HaveArgs) =>
-  ((member2.parameters[member2.parameters.length - 1].isVarArg && member1.parameters.length >= member2.parameters.length)
+  ((lastParameter(member2)?.isVarArg && member1.parameters.length >= member2.parameters.length)
```

Comment:
How about instead of lastParameter we add a `hasVarArg` on Constructor and Method? It seems more reusable.

Reply:
:+1: